42_grml

```diff
@@ -130,4 +137,8 @@ ${grub_prep}
 EOF
 done
 
+if [ -n "$GRUB_SUBMENU" -a -n "$iso_list" ] ; then
```

Comment:
Please let's use:

```
if [-n "${GRUB_SUBMENU}" ] && [ -n "${iso_list}" ] ; then
```

instead (as [ foo -a bar ] is not well defined, see https://www.shellcheck.net/wiki/SC2166 for further info)

(I'm aware that our script isn't shellcheck ready, that's something on my todo list :-/)